content/android/_index.md

```diff
@@ -11,4 +11,8 @@ listdescription: ""
 
 ### Overview
 
-Some overview on the Android project...
+WorldWind Android is a 3D virtual globe API for Android. You can use it to provide a geographic context, complete with terrain, for visualizing geographic or geo-located information.
+
+WorldWind Android provides high-resolution terrain and imagery, retrieved from remote servers automatically as needed. You can also provide your own terrain and imagery. WorldWind Android additionally provides a rich collection of shapes that you can use to represent information on the globe or in space.
+
+The WorldWind Android client is compatible to [API level](https://source.android.com/source/build-numbers) 19 (KitKat 4.4) and up. It displays the globe via the [GLSurfaceView](https://developer.android.com/reference/android/opengl/GLSurfaceView.html). Read more about how to integrate WorldWind into your app in the [Get Started](/android/get-started/) section. See how to implement basic functionality in the [Tutorials](/android/tutorials/) and dive into more topics in the [Developer Guide](/android/developer-guide/).
```

Comment:
The [project navigation bar](https://github.com/NASAWorldWind/NASAWorldWind.github.io/blob/develop/layouts/partials/project-navbar.html) includes links to Tutorials and Downloads. Do you think these should disappear until content is complete?

I'd like to keep the same html template between the projects if possible, but a customized version for each one isn't terribly complex.